data_safe_haven/external/api/graph_api.py

```diff
@@ -470,7 +470,7 @@ def create_user(
         email_address: str,
         phone_number: str,
     ) -> None:
-        """Create an AzureAD user if it does not already exist
+        """Create an Entra ID user if it does not already exist
```

Comment:
```suggestion
        """Create an Entra user if it does not already exist
```

Again I suspect that this should just be `Entra user`